centreon/www/class/centreon-clapi/centreon.Config.Poller.class.php

```diff
@@ -98,39 +98,48 @@ public function __construct($centreon_path, \Pimple\Container $dependencyInjecto
     }
 
     /**
-     * Get dynamic centcore pipe file when possible
+     * Write command to centcore pipe, using the dynamic centcore pipe file
+     * when possible
      *
-     * @return string
+     * @param string $cmd
+     * @param int $id
+     * @return int
      */
-    private function getCentcorePipe(): string
+    private function writeToCentcorePipe($cmd, $id): int
```

Comment:
As Centcore has been replaced by Gorgone few years ago did this method still relevant ?